### PR TITLE
Add workspace news item

### PR DESCRIPTION
**Pull request by nicknytko**
image

Please make any changes as needed.

### CODE REVIEW

**Comment by mwest1066** [comment]
Could we also include screenshots of VS Code and Xterm.js? I think these images are very powerful.

And I think we can how include direct links to XC 101 questions, like we do in the docs? So we could add a sentence like "Try it out yourself for [VS Code], [JupyterLab], or [Xterm.js]." where we link to the course (not course instance) questions?

File: news_items/017_workspaces/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p class="content">
+    Thanks to the amazing work by <a href="https://github.com/tdy">Tim Yang</a>, <a href="https://github.com/nicknytko">Nicolas Nytko</a>, and <a href="https://github.com/polohan">Yipeng Han</a>, PrairieLearn now supports <i>container workspaces</i> allowing students to run browser based applications such as Visual Studio Code and JupyterLab from inside PrairieLearn questions!  These workspaces are tightly integrated with the existing external grading system, meaning your existing graders can be used to autograde workspace files.  Workspaces are also synced and backed up, so a student's environment will be restored the next time they open the same workspace.
```
Comment (mwest1066) [inline news_items/017_workspaces/index.html:33]:
```suggestion
    Thanks to the amazing work by <a href="https://github.com/tdy">Tim Yang</a>, <a href="https://github.com/nicknytko">Nicolas Nytko</a>, <a href="https://github.com/polohan">Yipeng Han</a>, and <a href="https://github.com/nwalters512">Nathan Walters</a>, PrairieLearn now supports <i>container workspaces</i> that allow students to run browser based applications such as Visual Studio Code and JupyterLab from inside PrairieLearn questions!  These workspaces are tightly integrated with the existing external grading system, meaning your existing graders can be used to autograde workspace files.  Workspaces are also synced and backed up, so a student's environment will be restored the next time they open the same workspace.
```

Comment (nicknytko) [reply news_items/017_workspaces/index.html:33]: how could i have forgotten!! :fearful:

File: news_items/017_workspaces/info.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "uuid": "33f5fec9-e21c-422f-8ad2-b0d309bd72e6",
+    "title": "New PrairieLearn Workspaces",
```
Comment (tdy) [inline news_items/017_workspaces/info.json:3]:
```suggestion
    "title": "New PrairieLearn workspaces",
```
Looks like other news titles are sentence case. Or do we want to go ahead and brand "PL Workspaces" as a proper noun?

**Pull request merged**